### PR TITLE
[mlvm] Add cuDNN to documentation and update libraries

### PR DESCRIPTION
**Pull request by bradmiro**
Adding cuDNN to mlvm documentation, updating libraries.

Running into the same HDFS issue as in https://github.com/GoogleCloudDataproc/initialization-actions/pull/858#issuecomment-768583503

### CODE REVIEW

File: mlvm/mlvm.sh [hunk 1]
```diff
@@ -31,48 +31,54 @@ readonly INIT_ACTIONS_DIR=$(mktemp -d -t dataproc-init-actions-XXXX)
 readonly RAPIDS_RUNTIME="$(/usr/share/google/get_metadata_value attributes/rapids-runtime || echo "")"
 readonly INCLUDE_GPUS="$(/usr/share/google/get_metadata_value attributes/include-gpus || echo "")"
 readonly SPARK_BIGQUERY_VERSION="$(/usr/share/google/get_metadata_value attributes/spark-bigquery-connector-version ||
-  echo "0.17.0")"
+  echo "0.18.0")"
```
Comment (medb) [inline mlvm/mlvm.sh:34]: Use `0.18.1`

File: mlvm/mlvm.sh [hunk 2]
```diff
@@ -218,9 +226,6 @@ function main() {
   # Install Pip packages
   echo "Installing Pip Packages"
   install_pip_packages
-
-  # Clean up environment
-  conda clean -y --all
```
Comment (medb) [inline mlvm/mlvm.sh:223]: Why this is not necessary?

**Review by medb — CHANGES_REQUESTED** [review]

File: mlvm/mlvm.sh (hunk 2 above)
Comment (bradmiro) [reply mlvm/mlvm.sh:223]: I moved it here as cleaning up with Mamba is much faster.

https://github.com/GoogleCloudDataproc/initialization-actions/blob/94305303a69eec86a5af395b627cf8d5b0d7d831/mlvm/mlvm.sh#L148-L149

**Review by medb — CHANGES_REQUESTED** [review]
2.0 image tests are failing:

```
INFO: From Testing //mlvm:test_mlvm (shard 7 of 7):
==================== Test output for //mlvm:test_mlvm (shard 7 of 7):
Running tests under Python 3.8.7: /usr/bin/python3
[ RUN      ] MLVMTestCase.test_mlvm_gpu('STANDARD', 'standalone', 'DASK')
[  FAILED  ] MLVMTestCase.test_mlvm_gpu('STANDARD', 'standalone', 'DASK')
======================================================================
FAIL: test_mlvm_gpu('STANDARD', 'standalone', 'DASK') (__main__.MLVMTestCase)
test_mlvm_gpu('STANDARD', 'standalone', 'DASK') (__main__.MLVMTestCase)
test_mlvm_gpu('STANDARD', 'standalone', 'DASK')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/io_abseil_py/absl/testing/parameterized.py", line 265, in bound_param_test
    test_method(self, *testcase_params)
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/__main__/mlvm/test_mlvm.py", line 155, in test_mlvm_gpu
    metadata=metadata)
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/__main__/mlvm/test_mlvm.py", line 80, in verify_all
    self.verify_python()
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/__main__/mlvm/test_mlvm.py", line 27, in verify_python
    self.assert_dataproc_job(
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/__main__/integration_tests/dataproc_test_case.py", line 268, in assert_dataproc_job
    ret_code, stdout, stderr = self.assert_command(
  File "/usr/local/google/home/idv/.cache/bazel/_bazel_idv/1d68133c90d4317d3f3ab83fdd37b524/execroot/__main__/bazel-out/k8-fastbuild/bin/mlvm/test_mlvm.runfiles/__main__/integration_tests/dataproc_test_case.py", line 292, in assert_command
    self.assertEqual(
AssertionError: 1 != 0 : Failed to execute command:
gcloud dataproc jobs submit pyspark --cluster=test-mlvm-standard-2-0-20210216-062753-x10d --region=us-central1 gs://dataproc-init-actions-test-googcomhadoop-cloud-dev/20210216-062748-wsex/mlvm/scripts/python_packages.py
STDOUT:

STDERR:
Job [d504b2c709fc4ce9aef7df15c4a3edea] submitted.
Waiting for job output...
2021-02-16 07:06:57.126206: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
fc-list: error while loading shared libraries: libicuuc.so.68: cannot open shared object file: No such file or directory
Matplotlib is building the font cache; this may take a moment.

Segmentation fault: 11

Stack trace:
  [bt] (0) /opt/conda/default/lib/python3.8/site-packages/mxnet/libmxnet.so(+0x3c27360) [0x7f06479a7360]
ERROR: (gcloud.dataproc.jobs.submit.pyspark) Job [d504b2c709fc4ce9aef7df15c4a3edea] failed with error:
Google Cloud Dataproc Agent reports job failure. If logs are available, they can be found at:
https://console.cloud.google.com/dataproc/jobs/d504b2c709fc4ce9aef7df15c4a3edea?project=google.com:hadoop-cloud-dev&region=us-central1
gcloud dataproc jobs wait 'd504b2c709fc4ce9aef7df15c4a3edea' --region 'us-central1' --project 'google.com:hadoop-cloud-dev'
https://console.cloud.google.com/storage/browser/dataproc-0f21b6e8-c4c9-41c0-88d9-24ef341a339f-us-central1/google-cloud-dataproc-metainfo/760a880c-e242-4858-83b5-215898843d67/jobs/d504b2c709fc4ce9aef7df15c4a3edea/
gs://dataproc-0f21b6e8-c4c9-41c0-88d9-24ef341a339f-us-central1/google-cloud-dataproc-metainfo/760a880c-e242-4858-83b5-215898843d67/jobs/d504b2c709fc4ce9aef7df15c4a3edea/driveroutput


----------------------------------------------------------------------
Ran 1 test in 2364.961s

FAILED (failures=1)
```

**Comment by bradmiro** [comment]
@mengdong do you know when RAPIDS 0.18 will be released? There's currently a dependency conflict with cuSpatial 0.17 and the libraries on our base image. The conflict disappeared when I tried the nightly build of 0.18

**Comment by medb** [comment]
Please, rebase this branch on master HEAD

**Comment by bradmiro** [comment]
@medb rebased

**Review by medb — APPROVED** [review]

**Pull request merged**